core/platform/android/libaxmol/axutils.gradle

```diff
@@ -97,7 +97,7 @@ class axutils {
 //    }
 
     @SuppressWarnings('unused')
-    static String[] findNativeBuildTools(project, cmakeVer = "3.22.1+", ndkVer = "23.2.8568313") {
+    static String[] findNativeBuildTools(project, cmakeVer = "3.22.1+", ndkVer = "26.1.10909125") {
```

Comment:
should also change api leve in all gradle projects

Reply:
Thank you. API changed to 21. 
I did it once https://github.com/axmolengine/axmol/commit/0585fe563a800cb32a4501ee30048e79b8df4765 but it was reverted

Reply:
> Thank you. API changed to 21. I did it once [0585fe5](https://github.com/axmolengine/axmol/commit/0585fe563a800cb32a4501ee30048e79b8df4765) but it was reverted

yes, because when i build apk, can't install to android 4.2 emulator for testing GLESv2 compatibility.